### PR TITLE
fix: replace faSignInAlt with paragon icon

### PR DESCRIPTION
**Pull request by attiyaIshaque**
### Description

Currently, the faSignInAlt icon from fontawesome is being used in authn, but we want to replace it with a Paragon icon to provide a more consistent experience.

#### JIRA
https://2u-internal.atlassian.net/browse/VAN-1375

#### How Has This Been Tested?

Please describe in detail how you tested your changes.

#### Screenshots/sandbox (optional):

Include a link to the sandbox for design changes or screenshot for before and after. **Remove this section if its not applicable.**

|Before|After|
|-------|-----|
|      |      |

#### Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Sandbox, if applicable.
* [ ] Is there adequate test coverage for your changes?

#### Post-merge Checklist

* [ ] Deploy the changes to prod after verifying on stage or ask **@openedx/vanguards** to do it. 
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: src/common-components/SocialAuthProviders.jsx [hunk 1]
```diff
@@ -36,9 +37,11 @@ const SocialAuthProviders = (props) => {
       )
         : (
           <div className="font-container" aria-hidden="true">
-            <FontAwesomeIcon
-              icon={SUPPORTED_ICON_CLASSES.includes(provider.iconClass) ? ['fab', provider.iconClass] : faSignInAlt}
-            />
+            {SUPPORTED_ICON_CLASSES.includes(provider.iconClass) ? (
+              <FontAwesomeIcon icon={['fab', provider.iconClass]} />)
+              : (
+                <Icon className="h-75" src={Login} />
```
Comment (mubbsharanwar) [inline src/common-components/SocialAuthProviders.jsx:43]: Can we manage the icon position by reducing the padding from `font-container` instead using `h-75`?

**Review by mubbsharanwar — COMMENTED** [review]
Just one question otherwise looks good.

File: src/common-components/SocialAuthProviders.jsx (hunk 1 above)
Comment (attiyaIshaque) [reply src/common-components/SocialAuthProviders.jsx:43]: I have tried this but it also changes the position of `FontAwesomeIcon` which is placed inside the `font-container`. and also only the new Icon which is added needs to be fixed.

**Comment by shahbaz-arbisoft** [comment]
LGTM.

File: src/register/tests/RegistrationPage.test.jsx [hunk 2]
```diff
@@ -864,6 +864,27 @@ describe('RegistrationPage', () => {
...
+          thirdPartyAuthContext: {
+            ...initialState.commonComponents.thirdPartyAuthContext,
+            providers: [ssoProvider],
+          },
+          thirdPartyAuthApiStatus: COMPLETE_STATE,
+        },
+      });
+
+      delete window.location;
+      window.location = { href: getConfig().BASE_URL.concat('/login'), search: `?next=/dashboard&tpa_hint=${ssoProvider.id}` };
```
Comment (mubbsharanwar) [inline src/register/tests/RegistrationPage.test.jsx:882]: Should it (`/login`) not be `/register`? And also import the register URL from the constant file.

Comment (attiyaIshaque) [reply src/register/tests/RegistrationPage.test.jsx]: Addressed the comment.

**Review by mubbsharanwar — APPROVED** [review]

**Pull request merged**